plugins/generator-1.20.1/forge-1.20.1/mappings/attributes.yaml

```diff
@@ -0,0 +1,20 @@
+_default: Attributes.ARMOR
+ARMOR: Attributes.ARMOR

```

Comment:
Please sort this list in accordance to order of the vanilla file, as it makes porting later much easier. Start with fields from Attributes and then from ForgeMod